### PR TITLE
Whatsnew 76 [Fixes #8703]

### LINKED ISSUE #8703: Code - WNP 76
[Releng bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1628904)

### PR DESCRIPTION
**Pull request by Errolyn**
## Description

Create whatsnew-76 page

## Issue / Bugzilla link
 #8703

## Testing
http://localhost:8000/en-US/firefox/76.0/whatsnew/all/. -- should have 3 card items at the bottom
http://localhost:8000/zh-TW/firefox/76.0/whatsnew/all/ -- should have 2, facebook container should not be present.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success up-to-date">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-c-emphasis-box">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html:34]: Nit: this emphasis box should probably be inside a `<div class="mzp-l-content"></div>` container?. Without it it runs flush to the viewport at smaller screen sizes.

File: media/js/firefox/whatsnew/whatsnew-76.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof window.Mozilla === 'undefined') {
```
Comment (alexgibson) [inline media/js/firefox/whatsnew/whatsnew-76.js:6]: No need for a namespace declaration here?

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html:5]: I'm not sure these macros are needed in the template?

**Comment by Errolyn** [comment]
I couldn't think of a reason the main cta wouldn't show up, so instead of writing a test to check if its there I wrote a couple of tests for making sure the page is displaying/non-displaying fb-container content correctly. Let me know if there was something you were thinking about when asking for the cta test.

Thanks for the review!

**Comment by alexgibson** [comment]
> I couldn't think of a reason the main cta wouldn't show up

Some possible examples: 

- The page throws a 500 error.
- A CSS change inadvertently hides the button.
- Another element inadvertently obscures visibility of the button.

> so instead of writing a test to check if its there I wrote a couple of tests for making sure the page is displaying/non-displaying fb-container content correctly.

This is a good test to have, but it's reliant on the `zh-TW` being active in production, and it looks like strings have not yet been exposed. As such, we can't merge like this without it failing on staging. It's also a lesser critical part of the page, so I'd probably advocate for testing the main CTA for now. We can always add a locale specific test later.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+{% block page_title %}{{ _('What’s new with Firefox') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_76') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew-76' %}
+{% set _utm_campaign = 'whatsnew-76' %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html:18]: I don't believe these variables are needed any more, right?

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+              <img src="{{ static('img/logos/fbcontainer/logo-fbcontainer.svg') }}" width="48px" alt="">
+            </div>
+
+            <h3 class="c-picto-block-title">{{ _('Facebook Container') }}</h3>
+            <div class="c-picto-block-body">
+              <p>
+                {{ _('Stay connected to the people you can’t be with in real life, and stop Facebook from following you online.') }}
+              </p>
+              <br>
+              <a href="https://addons.mozilla.org/en-US/firefox/addon/facebook-container/?{{ utm_params }}" class="picto-link">{{ _('Add Facebook Container') }}</a>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx76.html:91]: Remove locale
```suggestion
              <a href="https://addons.mozilla.org/firefox/addon/facebook-container/?{{ utm_params }}" class="picto-link">{{ _('Add Facebook Container') }}</a>
```

**Pull request merged**